torchvision/csrc/ROIAlign.h

```diff
@@ -2,13 +2,8 @@
 
 #include "cpu/vision_cpu.h"
 
-#ifdef WITH_CUDA
-#include "autocast.h"
-#include "cuda/vision_cuda.h"
-#endif
-#ifdef WITH_HIP
+#if defined(WITH_CUDA) || defined(WITH_HIP)
 #include "autocast.h"
-#include "hip/vision_cuda.h"
```

Comment:
I was actually also referring to `"cuda/vision_cuda.h"` and `"hip/vision_cuda.h"`. But as your editor said, this is not fundamental because everything is a header in here.

I was maybe thinking that we could put this repetitive includes in a separate header file which contains only these headers? or maybe just remove this altogether?

Reply:
As discussed on chat, I'll restore the includes for all Dispatcher PRs and clean up this bit of the code on a separate PR.